src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,19 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = { items: [] };
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStatus(state) {
+      // although the immer library is available there is a linter rule for param-reassignment
+      return {
+        ...state,
+        items: 'Under construction',
+      };
+    },
```

Comment:
- [ ] Please update the categories slice for the check status action it should return the string "Under Construction"  and In case of unknown action, it should return the current state as requested in the project requirements.
  - You can consider the below code.
```javascript
 checkStatus: (state, action) => {
      state.items =
        action.payload === "Under construction"
          ? "Under construction"
          : state.items
    },
```

Reply:
Or maybe I should ignore this Eslint report because I should stick to the toolkit way of doing things 🤔.